lib/lita-reviewme/github.rb

```diff
@@ -0,0 +1,66 @@
+require 'delegate'
+
+module Lita
+  module Reviewme
+    class Github
+      extend Forwardable
+
+      attr_reader :config, :repo, :pr_id
+
+      delegate [
+        :github_access_token,
+        :github_comment_template,
+        :github_comment,
+        :github_request_review
+      ] => :config
+
+      def initialize(config, repo, pr_id)
+        @config = config
+        @repo = repo
+        @pr_id = pr_id
+      end
+
+      def client
+        @client ||= Octokit::Client.new(access_token: github_access_token)
+      end
+
+      def pull_request
+        @pull_request ||= client.pull_request(repo, pr_id)
+      end
+
+      def owner
+        pull_request.user.login
+      rescue Octokit::Error
+        raise UnknownOwner
+      end
+
+      def assign_and_or_comment(reviewer)
+        request_review(reviewer) if github_request_review
+        add_comment(reviewer) if github_comment
+      rescue Octokit::Error => e
+        puts e
+        raise CannotPostComment
+      end
+
+      private
+
+      def request_review(reviewer)
+        options = { accept: 'application/vnd.github.black-cat-preview' }
```

Comment:
That's correct, apparently [Github releases new API's as "previews"](https://github.com/octokit/octokit.rb/blob/8c6a9106092bae59c804386cda64d05156512de3/lib/octokit/preview.rb#L18) and to call them you need to explicitly pass a custom media type.